spec/features/display_status_spec.rb

```diff
@@ -1,6 +1,8 @@
 describe "Display status text" do
   let(:proposal) { FactoryGirl.create(:proposal, :with_parallel_approvers) }
   before do
+    proposal.approvers.first.update(first_name: "Uniquely", last_name: "Named")
+    proposal.approvers.second.update(first_name: "Onlyof", last_name: "Itskind")
```

Comment:
Yes (the "lino" name wasn't too unique); I'll update the relevant tests to make it more apparent how this is used